pkg/connectorbuilder/connectorbuilder.go

```diff
@@ -325,25 +325,36 @@ func (b *builderImpl) ListTicketSchemas(ctx context.Context, request *v2.Tickets
 		return nil, fmt.Errorf("error: ticket manager not implemented")
 	}
 
-	out, nextPageToken, annos, err := b.ticketManager.ListTicketSchemas(ctx, &pagination.Token{
-		Size:  int(request.PageSize),
-		Token: request.PageToken,
+	retryer := retry.NewRetryer(ctx, retry.RetryConfig{
+		MaxAttempts:  0,
+		InitialDelay: 15 * time.Second,
+		MaxDelay:     0,
```

Comment:
@laurenleach I know there was some concern about having `MaxAttempts` and `MaxDelay` without a limit. I had it this way to test it out. Do we want to stick with it here for now?

Reply:
I was more worried about Validate, @ggreer do you have any concerns?